.github/workflows/build.yml

```diff
@@ -5,22 +5,22 @@ concurrency:
   cancel-in-progress: true
 jobs:
   cocoapods:
-    name: CocoaPods (Xcode 14.3)
+    name: CocoaPods (Xcode 15.1)
```

Comment:
Looks like the base branch needs to be updated for this PR

Reply:
That's strange... I'll update the `app-installed-check` branch with the main and see if that helps